proteus/tests/SWFlow/comparison_files/comparison_solitary_island_h_t2.csv

```diff
@@ -1 +1 @@
64805556443915,0.3289832291657706,0.34669735455162254,0.3741784110708187,0.36993042665061693,0.34256660166141784,0.32670468811302644,0.3221050407533982,0.32048224486190735,0.3201502800177186,0.3200297944455208,0.32000831249798034,0.31999853472387485,0.31999896559113666,0.319997598664818,0.31999934769077965,0.319998789369939,0.31999978583570854,0.3199995148113594,0.31999995557239536,0.31999985593735775,0.31999999699749104,0.319999967047009,0.3200000009064484,0.3199999937580036,0.32000000037484044,0.3199999989638499,0.32000000008548934,0.3199999998427382,0.3200000000157588,0.31999999997464845,0.32000000000502354,0.3199999881826628,0.32000003962795365,0.32000019070262065,0.320000537672133,0.3200033399661638,0.3200077999504305,0.3200508906612851,0.3201297095585882,0.32070955324575673,0.3222902829779073,0.3288662105260887,0.34843893228276523,0.3735015438785987,0.36759938087595045,0.3430299937766018,0.32732118532973337,0.3218954570982847,0.3205620228172295,0.3201369012930779,0.32004077595571934,0.3200110495981492,0.320003573198102,0.3200018085120484,0.3200006503689089,0.32000067343044786,0.32000017042351914,0.3200002866849807,0.3200000428271821,0.3200001003020517,0.32000000712431104,0.32000002754328505,0.32000000034191833,0.3200000061465277,0.3199999998242774,0.3200000011727324,0.3199999999328054,0.3200000001995761,0.31999999998439843,0.3200000000318412,0.31999999999661766,0.3200000000092956,0.31999997234012834,0.3200001240916099,0.320000039020681,0.3200014379731562,0.3200011552314324,0.3200176080538865,0.32002542004138806,0.32021742035338735,0.32051380622722975,0.32264892125605465,0.3289814817677178,0.3467064041789896,0.37419434672501833,0.36993092011226547,0.34256273031066764,0.3267039324545287,0.3221061093758065,0.3204845108858514,0.3201524114200517,0.3200336955985038,0.3200105107793427,0.32000195410340865,0.3200005341073283,0.31999984235976126,0.31999995038139656,0.3199998456927162,0.31999996902126726,0.3199999329301024,0.31999999186049055,0.31999997802577995,0.3199999987468589,0.31999999420688047,0.3199999999516725,0.3199999987170015,0.32000000003477125,0.31999999975165094,0.3200000000131761,0.3199999999564599,0.32000000000323053,0.3199999999918939,0.32000000000125983,0.3199999889015452,0.32000003847261793,0.32000019206298447,0.3200005334323018,0.3200033299924424,0.3200078400916868,0.3200505631070416,0.32013090322752835,0.3207058631679516,0.32230491606901057,0.3288677307962416,0.3484757300065728,0.37343314876624617,0.36761792111885605,0.34303865071270895,0.32732430169006754,0.3219002770579276,0.3205613888393235,0.3201359684819116,0.32004018263465006,0.32000987868383135,0.3200030196256361,0.3200009169840541,0.32000030873233276,0.32000019656782536,0.32000005562445977,0.3200000719145213,0.3200000131227546,0.3200000255392641,0.32000000264086215,0.3200000076179129,0.3200000003398002,0.3200000019028069,0.32000000000261386,0.3200000004092204,0.3199999999875415,0.32000000007816254,0.3199999999957525,0.3200000000138391,0.3199999999988272,0.3200000000044532
```

Comment:
maybe lfs this too? @cekees should we just move all .csv to git lfs moving forward?

Reply:
Yes, my vote is to push them all to git lfs. It hasn't been that much of an issue to use lfs even on the hpc machines, and I think we still frequently have need to store non-source-code items in the repository.